### DISCUSSION TITLE
ETA

### DISCUSSION THREAD

**Discussion by pineapplee0** [skylerspaeth/SkyOS#5] [Q&A] [answered]
Hello SkyOS nation!! I was wondering what the ETA on this project was so I could setup and use SkyOS for myself! Can't wait to see more updates :)

**Comment by skylerspaeth** [comment]
😀 🔜

**Comment by skylerspaeth** [comment, marked as answer]
Within the week hopefully!

**Comment by skylerspaeth** [comment]
Thanks for your enthusiasm and support!
